core/templates/feedback.html

```diff
@@ -0,0 +1,46 @@
+{% extends "layout.html" %}
+
+{% load i18n %}
+{% load bootstrap3 %}
+{% load account %}
+
+{% load deck_tags %}
+
+{% block head_title %}
+  {% trans "Feedback" %}
+{% endblock %}
+
+{% block head %}
+
+{% endblock head %}
+
+{% block content %}
+  <div class="row header">
+    <h2>
+      {% trans "Feedback" %}
+    </h2>
+  </div>
+  <div class="row form-wrapper">
+    <div class="column">
```

Comment:
You could add a message somewhere in this template with something like:

> You can also chat with us on Gitter.im

What do you think?